email_alert_service/models/change_history.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeHistory
+  attr_reader :history
+
+  def initialize(history:)
+    @history = history
+  end
+
+  def latest_change_note
+    change_note = history&.first
```

Comment:
I don't know if it's always guaranteed to be the first entry here, I would feel more confident if we ordered by the dates first.

Reply:
Hmm, although now I see this is what `EmailAlertTemplate` used to do and I guess there haven't been any problems.

Reply:
yeah.. I thought that too but thought it best to just copy the carry functionality over for now.